index.html

```diff
@@ -6221,7 +6221,9 @@ <h3>Set #1: The DID subject <em>is</em> the DID controller</h3>
 From a graph model perspective, even though the nodes identified as the
 <a>DID controller</a> and <a>DID subject</a> in figure 4 are distinct,
 there is a logical arc connecting them to express a semantic equivalence
-relationship (in RDF/OWL, this is expressed using the <a href="https://www.w3.org/TR/owl-ref/#sameAs-def"><code>owl:sameAs</code> predicate)</a>.
+relationship (in RDF/OWL, this is expressed using the <a
+href="https://www.w3.org/TR/owl-ref/#sameAs-def">
+<code>owl:sameAs</code> predicate)</a>.
```

Comment:
"and then I read the text..."
```suggestion
relationship. (In RDF/OWL, this is expressed using the <a
href="https://www.w3.org/TR/owl-ref/#sameAs-def">
<code>owl:sameAs</code> predicate</a>.)
```